app/src/main/res/menu/manage_group_menu.xml

```diff
@@ -0,0 +1,10 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context="protect.card_locker.ManageGroupActivity">
+    <item
+        android:id="@+id/action_confirm"
+        android:icon="@drawable/ic_done"
+        android:title="@string/action_search"
+        app:showAsAction="always|collapseActionView" />
+</menu>
```

Comment:
Android design guidelines state that you shouldn't have a menu to confirm, but use a FloatingActionButton. Look at the other activities (Main, LoyaltyCardView, LoyaltyCardEdit, Groups) how that is created